### PR TITLE
prepared ccd config for recovering missing documents

### PR DESCRIPTION
**Pull request by yarpo**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DIV-5852

### Change description ###
https://tools.hmcts.net/confluence/display/RCCD/Instruction+for+Services+to+recover+missing+Documents

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: definitions/divorce/json/AuthorisationCaseEvent.json [hunk 1]
```diff
@@ -5542,5 +5542,54 @@
...
+    "CaseEventID": "SYSTEM_MAINTENANCE",
+    "UserRole": "caseworker-divorce-systemupdate",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "SYSTEM_MAINTENANCE",
+    "UserRole": "citizen",
+    "CRUD": "R"
```
Comment (alexwakeman) [inline definitions/divorce/json/AuthorisationCaseEvent.json]: Not sure we need to give citizen `R` access..?

**Comment by hosainnet** [comment]
Conflicts

File: definitions/divorce/json/AuthorisationCaseEvent.json (hunk 1 above)
Comment (yarpo) [reply definitions/divorce/json/AuthorisationCaseEvent.json]: They requested to add this to ALL roles.

**Comment by tchow8** [comment]
who is meant to trigger the SYSTEM_MAINTENANCE event? superuser doesn't seem to have permissions on it?

**Comment by hosainnet** [comment]
> who is meant to trigger the SYSTEM_MAINTENANCE event? superuser doesn't seem to have permissions on it?

I think it's some backend/java task in CCD

**Comment by tchow8** [comment]
Sorry just confused, since I don't see a single U permission on it. Which means even if some backend task triggers it it'll return 403?

**Comment by hosainnet** [comment]
> Sorry just confused, since I don't see a single U permission on it. Which means even if some backend task triggers it it'll return 403?

@yarpo ?

**Pull request merged**